peering/models/models.py

```diff
@@ -917,6 +919,40 @@ def get_absolute_url(self):
             "peering:internetexchangepeeringsession_details", kwargs={"pk": self.pk}
         )
 
+    def merged_export_policies(self, reverse=False):
```

Comment:
noting both `merged_export_policies` and `merged_import_policies` are used under both `DirectPeeringSession` and `InternetExchangePeeringSession`, perhaps we could abstract this under `BGPSession`? Only a small amount of logic is different.